### PR TITLE
Implementing Stylist UI Script

### PR DESCRIPTION
**Pull request by JohnnyPlayy**
* **Addressed Issue(s)**: 

* **Server Mode**: Pre-Renewal / Renewal

* **Description of Pull Request**: 

 Thanks to Lemongrass!

TODOs:

- [x] Test and Check NPCs

- [x] Translate the script

### CODE REVIEW

File: npc/re/merchants/Extended_Stylist.txt [hunk 1]
```diff
@@ -2,108 +2,109 @@
...
 }
 
 //= Stylist of Lasagna
 //===================================================
-lasagna,135,110,3	script	스타일리스트#lasagna	4_M_MERCAT2,{
-	mes "[롱카]";
-	mes "패션에 살고 패션에 죽는 도람이라면 나 롱카의 위대한 손길에 몸을 맡겨봐~♬";
-	switch(select("새롭게 스타일을 변경한다:대화를 마친다")) {
+lasagna,134,113,3	script	Stylist#lasagna	4_M_MERCAT2,{
+	mes "[Lonza]";
```
Comment (Everade) [inline npc/re/merchants/Extended_Stylist.txt:67]: [Longka]

**Review by Everade — COMMENTED** [review]
Some more accurate translations. Looks good overall though.

File: npc/re/merchants/Extended_Stylist.txt [hunk 2]
```diff
@@ -2,108 +2,109 @@
...
 }
 
 //= Stylist of Prontera
 //===================================================
-prt_in,243,168,4	script	스타일리스트#prontera	4_F_02,{
-	mes "[요보비치]";
-	mes "어서오세요. 프론테라의 스타일리스트 명인 요보비치입니다. 무엇을 도와드릴까요?";
-	switch(select("새롭게 스타일을 변경한다:대화를 마친다")) {
+prt_in,243,168,4	script	Stylist#prontera	4_F_02,{
+	mes "[Jovovich]";
```
[the remark this answers is not in the document]
Comment (Haydrich) [reply npc/re/merchants/Extended_Stylist.txt:31]: > 요보비치

actually Jovovich is more prefer for a female 👯‍♀️

File: npc/pre-re/warps/cities/alberta.txt [hunk 3]
```diff
@@ -0,0 +1,10 @@
+//===== rAthena Script ======================================= 
+//= Alberta City Portals
+//===== Changelogs: ==========================================
+//= 1.0 First Version. [JohnnyPlayy]
+//============================================================
+
+//= Portal - ( Rooms )
+//============================================================
+alberta,134,38,0	warp	Room_Entrance#alberta-01	1,1,alberta_in,70,141
+alberta_in,73,141,0	warp	Room_Exit#alberta-01	1,1,alberta,137,37
```
Comment (Atemo) [inline npc/pre-re/warps/cities/alberta.txt:10]: The change of name was intended ?
```suggestion
alberta,134,38,0	warp	alb02	1,1,alberta_in,70,141
alberta_in,73,141,0	warp	alb02-1	1,1,alberta,137,37
```

File: npc/re/merchants/Extended_Stylist.txt (hunk 1 above)
Comment (JohnnyPlayy) [reply npc/re/merchants/Extended_Stylist.txt:67]: This name was based on the translation of iro!

File: npc/re/merchants/Extended_Stylist.txt (hunk 2 above)
[the remark this answers is not in the document]
Comment (JohnnyPlayy) [reply npc/re/merchants/Extended_Stylist.txt:31]: I study 5 languages, I can sometimes get a little confused. And I'm pretty sure that name I translate well. And still follow the iro concept!

File: npc/pre-re/warps/cities/alberta.txt (hunk 3 above)
Comment (JohnnyPlayy) [reply npc/pre-re/warps/cities/alberta.txt:10]: I wrote before looking at how the portals were. That's why it's like this.

File: npc/re/merchants/Extended_Stylist.txt (hunk 1 above)
Comment (Everade) [reply npc/re/merchants/Extended_Stylist.txt:67]: Then leave it this way 😄

**Review by RagnaWay — APPROVED** [review]
Merge!

**Comment by Lemongrass3110** [comment]
> Merge!

Dear developers of rAthena. Thank you for spending your time on this project.
Could you please be so kind and merge this pull request?

**Comment by RagnaWay** [comment]
> > Merge!
> 
> Dear developers of rAthena. Thank you for spending your time on this project. Could you please be so kind and merge this pull request?

Ah thank you! Now I have someone to speak for me! :P

**Pull request merged**